fsthttp/backend.go

```diff
@@ -245,40 +246,59 @@ func (b *BackendOptions) UseSSL(v bool) *BackendOptions {
 
 // SSLMinVersion sets the minimum allowed TLS version on SSL connections to this backend.
```

Comment:
```suggestion
// SSLMinVersion sets the minimum allowed TLS version on SSL connections to this backend.
// Setting this will enable SSL for the connection as a side effect.
```

Reply:
Updated all the doc comments and force-pushed.